hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/spark/sql/hudi/TestAlterTableDropPartition.scala

```diff
@@ -305,12 +317,15 @@ class TestAlterTableDropPartition extends TestHoodieSqlBase {
         assertResult(false)(existsPath(
           s"${tmp.getCanonicalPath}/$tableName/year=2021/month=10/day=01"))
 
+        // TODO (HUDI-3135): These validations are failing. Due to lazy deletion,
```

Comment:
@XuQianJin-Stars I've disabled some of these `show partitions` check as they were failing. Even though I've verified that cleaner will eventually delete the partition, we need to check why the fs view does not exclude the partition.

Reply:
> @XuQianJin-Stars I've disabled some of these `show partitions` check as they were failing. Even though I've verified that cleaner will eventually delete the partition, we need to check why the fs view does not exclude the partition.

Let me check it.

Reply:
@XuQianJin-Stars Thanks for fixing it!